### DISCUSSION TITLE
Idea: color variations in dark theme should become lighter not darker

### DISCUSSION THREAD

**Discussion by t1u1** [saadeghi/daisyui#1600] [Ideas / feature requests]
Color variations such as `base-100` being darker than the base makes sense for light themes. However for dark themes, I think their meaning should be flipped, so that they have the same effect in terms of accent.

For example, look at the `forest` and `coffee` themes. The accents and color contrast in many places is lost because the base color itself is dark. Making it darker doesn't yield a very distinct color that is set off against the base color.

**Comment by saadeghi** [comment]
It's not a good practice to simply invert the color shades for a dark theme. It's better to keep the same pattern of color shades for both themes. for example if we have a darker box on lighter background on light theme. we should also have the same pattern on the dark theme (darker box on lighter background) 

See these 3 examples: https://play.tailwindcss.com/FIsJ2GZDVz

As you can see, the second example looks better than 3rd one because it's not inverting the colors. The second example is applying the same order of color shade increment and it looks better.

However if you want `100/200/300` become lighter on each level, you can customize the colors as you want.  
See [theme generator](https://daisyui.com/theme-generator/) and [other optional color names you can customize](https://daisyui.com/docs/colors/)

daisyUI builtin themes like `forest` and `coffee` are opinionated themes. There are even themes like `lofi` which are basically black and white just to demonstrate the possibilities but I alway encourage people to customize the daisyUI colors and make their own unique theme. 

Let me know if you have any questions

**Reply by t1u1** [reply]
Thanks for your detailed response. The second example looks fine with the `dark` theme because the base of the dark theme isn't all that dark. However, for a theme which has a really dark base like `forest`, the right side call-out doesn't look distinguished at all.

See https://play.tailwindcss.com/NVNpYj4nP6

Customizing is an option, will explore it, thanks.

Perhaps, for now, the forest theme can define `100/200/300` to be lighter, for a good out-of-box experience.

Also, the documentation implies that `100/200/300` will be always darker. It could clarify that the effect depends on the theme. I initially thought that it is automatically made darker by some pre-processing, and was not customizable.

**Reply by saadeghi** [reply]
>I initially thought that it is automatically made darker by some pre-processing  

It is! `base-100` color automatically gets darker on `200` and `300` shades. because 200 and 300 are optional colors. 

>and was not customizable

You can set your desired values for them if you don't want that color generation.

All themes are opinionated at some point. If you take a look at all the design systems companies and websites are using in the wild, there are different approaches on how many colors and how many shades they use. For example, some design systems are just black and white. Some have 1 single brand color. Some of them use different surface/base color. Some of them have primary/secondary/accent, etc.  
daisyUI's `forest` theme in particular, is using an extremely dark `base` color so all the darker shades (`base-200` and `base-300`) are not recognizable from `base-100` but that won't make an accessibility issue (because the actual content is using a high contrast color: `base-content` and the content is readable), it's just fewer color choices for backgrounds.  
There are similar scenarios for a few other themes as well. For example, on `wireframe`,`black` or `lofi` themes there's no difference between `primary`, `secondary` and `accent` colors. They're all the same because these themes are examples of using fewer number of colors in a design system.

Creating your own theme is super easy tho. Give it a try: https://daisyui.com/theme-generator/

**Reply by t1u1** [reply]
Ok, Thanks, I will start customizing the themes.

A slightly off-topic suggestion: the theme generator could have a way to initialize the palette with one of the bundled themes. That will make it very easy for design-ignorant people like me to have a base palette to work with.
